pyvecorg/data/coc.yml

```diff
@@ -0,0 +1,51 @@
+heading:
+  cs: Code of Conduct
+  en: Code of Conduct
+
+text:
+  cs: >-
+      TBD
+  en: |
+    We value the participation of **each member** of the **Python community** and want all attendees to have an enjoyable and fulfilling experience. Accordingly, all attendees are expected to show **respect** and **courtesy** to other attendees throughout the event.
+    To make clear what is expected, all attendees, speakers, organizers, volunteers or other persons at any official PyCon CZ event are required to conform to the following Code of Conduct. Organizers will enforce this code throughout the event.
+
+    ## The Short Version
+    We are dedicated to providing a harassment-free event experience for everyone, regardless of gender, sexual orientation, physical appearance, body size, disability, race, ethnicity, religion, or otherwise. We do not tolerate harassment of event participants in any form, on-line or off-line.
+    All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate for any event venue, including talks.
+    Be kind to others. Do not insult or put down other attendees. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate at our events.
+    If a person engages in harassing behaviour, the organizers will take prompt action, including expulsion from the event without a refund. Thank you for helping make this a welcoming, friendly event for all.
+
+    ## The Longer Version
+    Harassment includes offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, offensive verbal comments, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention.
+    Participants asked to stop any harassing behaviour are expected to comply immediately.
+    Sponsor and vendor booths or similar activities are also subject to the anti-harassment policy. In particular, sponsors and vendors should not use sexualized images, activities, or other offensive material. Booth staff (including volunteers) should not wear sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment. No alcoholic beverages may be served at the booths.
+    Be careful with the words that you choose. Remember that sexist, racist, and other exclusionary jokes may be offensive to those around you. Excessive swearing and offensive jokes, excessive alcohol consumption, and use of illegal drugs are not appropriate at our events.
```

Comment:
1.  Na začátku:

    > Organizers will enforce this code throughout the event.

2.  Zde:

    > excessive alcohol consumption, and use of illegal drugs are not appropriate at our events

@aleszoulek Jsi si jistý, že je toto pražské (či jiné) Pyvo schopno reálně plnit?